externs/pictureinpicture.js

```diff
@@ -1,6 +1,6 @@
 /**
  * @license
- * Copyright 2016 Google Inc.
```

Comment:
For the reason that I don't fully understand, we're not allowed to ever update the year :/